packages/app/lib/app.dart

```diff
@@ -16,7 +17,7 @@ class MeliApp extends StatelessWidget {
 
     return GraphQLProvider(
         client: client,
-        child: MaterialApp.router(
+        child: MeliCameraProvider(MaterialApp.router(
```

Comment:
I've just stumpled upon `MultiProvider`, might be nicer to read:

```
MultiProvider(
  providers: [
    ChangeNotifierProvider(builder: (_) => FirstProvider()),
    ChangeNotifierProvider(builder: (_) => SecondProvider()),
  ],
  child: someWidget,
)
```

Reply:
I'll wait and see if we wanna move where the wrapping happens before changing this. I like the pattern in any case :+1:

Reply:
Unless I'm mistaken we'll need to adapt our current providers (which extend inherited widget) in order to use this. It is nice though, so happy for this refactor to happen in a follow-up step.